Resources/public/js/LineTimetable/select_stops.js

```diff
@@ -0,0 +1,210 @@
+define('mtt/line_timetable/select_stops', ['jquery', 'jquery_ui_sortable'], function($, sortable) {
+    var selectionChanged = false;
+    var underProgress = false;
+    var errorDiv = "<div class='alert alert-danger alert-dismissable danger'><span></span><button type='button' class='close' data-dismiss='alert' aria-hidden='true'>&times;</button></div>";
+
+    /**
+     * Saving selection event
+     */
+    $('.save-selection').on('click', function() {
+        _saveSelection();
+    });
+
+    /**
+     * Selecting a route event
+     */
+    $('#select-route').on('change', function() {
+        if (selectionChanged && confirm('Souhaitez-vous enregistrer les changements ?'))
```

Comment:
Hi, could you use bazinga module to make messages' translation possible? You can find an example here https://github.com/Tisseo/MttBundle/blob/feature-60-assign-stop-point-to-line-timetable/Resources/public/js/layout/validator.js